### ISSUE TITLE
Docs for `/events/{eventKey}/matches/{matchKey}/teams/{teamKey}/stats` are wrong

### ISSUE THREAD

**Issue by calebeby** [Pigmice2733/peregrine-backend#226]
The docs say the data is like this:

```json
[
  {
    "max": 4,
    "avg": 2.25,
    "name": "Rocket Hatches Lvl 1"
  }
]
```

But actually:

```json
{
  "summary": [
    {
      "max": 4,
      "avg": 2.25,
      "name": "Rocket Hatches Lvl 1"
    }
  ],
  "team": "..."
}
```

Also sometimes `summary` is `null`, is that expected or not?

**Comment by calebeby** [comment]
Also why does it return `max`/`avg`? This doesn't make a lot of sense to me :thinking:

**Comment by fharding1** [comment]
Fixed by 984bae5
